File: 第十次/10-2/Rectangle.cpp
#include"Rectangle.h"

Rectangle::Rectangle() {
	width = 1;
	height = 1;
}

Rectangle::Rectangle(double newWidth, double newHeight) {
	setWidth(newWidth);
	setHeight(newHeight);
}

double Rectangle::getWidth() {
	return width;
}
void Rectangle::setWidth(double newWidth) {
	width = newWidth;
}

double Rectangle::getHeight() {
	return height;
}
void Rectangle::setHeight(double newHeight) {
	height = newHeight;
}

double Rectangle::getArea() {
	return height * width;
}

File: Python/第11章/11-1/Rectangle.py
class Rectangle():
    def __init__(self, width, height):
        self.__width = width
        self.__height = height

    def setWidth(self):
        self.__width = width
    def setHeight(self):
        self.__height = height

    def getWidth(self):
        return self.__width
    def getHeight(self):
        return self.__height

    def getArea(self):
        return self.__width * self.__height

    def __str__(self):
        return str("width: {}, height: {}, area: {}".format(self.getWidth(), self.getHeight(), self.getArea()))